File: fixtures/mock-projects/calculator-app/go/calc.go
package main

// BUG: 故意语法错误 — 缺少闭合大括号
func Add(a int, b int) int {
	return a + b

// BUG: 除零未检查
func Divide(a int, b int) int {
	return a / b
}

File: fixtures/mock-projects/calculator-app/py/calc.py
# BUG: 语法错误 — 缺少冒号
def add(a, b)
    return a + b


def subtract(a, b):
    return a - b


# BUG: 变量名拼写错误
def multiply(a, b):
    result = a * b
    return resut  # typo: resut 应为 result


def divide(a, b):
    if b == 0:
        raise ValueError("Cannot divide by zero")
    return a / b
